### ISSUE TITLE
Account Line (Table)

### ISSUE THREAD

**Issue by tscottjendev** [Jersey-Electricity-plc/jwissues#54]
Do we need a description type field to distinguish lines other than line no? With having multiple utilities and the history of inactive lines it will quickly become very confusing to find the line to be looking for.  Maybe filter to only active by default?

**Comment by RPCJendev** [comment]
I don't think we do need a description but open to discuss. All the info is already on the line (tariff , address etc) and it is green (if we choose to keep colours). Inactive ones are different colour and still has all the info on.
